# Author: Alvaro Huanay a.huanay.de.dios@fz-juelich.de
# This patch solves the "name 'macro' is not defined" present in Bazel v7.7.0
diff --git a/MODULE.bazel b/MODULE.bazel
index 0a5ea5b..70aa431 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -12,6 +12,14 @@ module(
 # Bazel module dependencies
 # =========================================
 
+# Pin bazel_features to version compatible with Bazel 7.x
+bazel_dep(name = "bazel_features", version = "1.21.0")
+
+single_version_override(
+    module_name = "bazel_features",
+    version = "1.21.0",
+)
+
 bazel_dep(name = "rules_license", version = "0.0.7")
 bazel_dep(name = "bazel_skylib", version = "1.7.1")
 bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
